projects/igniteui-angular/src/lib/chips/chip.component.ts

```diff
@@ -156,10 +334,23 @@ export class IgxChipComponent implements AfterViewInit {
         }
     }
 
+    /**
+     * Sets the `IgxChipComponent` to be selected.
```

Comment:
Should be: "Returns if the `IgxChipComponent` is the last one among its siblings."